app/controllers/api_controller.rb

```diff
@@ -133,11 +133,11 @@ def version_config
   end
 
   def collection_config
-    Api::Config.collections
+    Api::Config.collections.to_hash
```

Comment:
@Fryguy I'll remove these `to_hash`s in a follow up - doing so here I thought might take this PR out of scope/hard to review